_docs/tasks/rate-limiting.md

```diff
@@ -101,10 +101,58 @@ The following configuration applies a `5qps` rate limit only to version `v3` of
    Since the `reviews-v3` service is unable to access `ratings` we stop seeing `red` stars on the UI.
 
 
-## Dimensioned rate limits [WIP]
+## Understanding rate limits
+In the preceding examples we saw how Mixer applies rate limits to requests that match certain conditions.
+
+Every distinct rate limit configuration represents a counter. When a request arrives, Mixer increments the counter that applies
+to the request. If the counter exceeds the limit specified by the `maxAmount` key, Mixer returns a `RESOURCE_EXHAUSTED` message 
+to the proxy which in turn returns status `HTTP 429` to the caller. The counter is periodically reset to 0 as specified 
+by the `expiration` duration. Multiple rate limits may apply to the same request. 
+
+Consider the following example
+```yaml
+descriptorName: RequestCount
+maxAmount: 5000
+expiration: 5s
+labels:
+  label1: target.service
+```
+This defines a counter with a limit of `5000` that resets every `5 seconds`.
```

Comment:
This will be the preferred approach I believe, since it avoids the '11th hour' problem where actual QPS can be twice as high for a given time interval than what is stated.